java/gazelle/resolve_test.go

```diff
@@ -63,10 +65,10 @@ java_library(
 java_library(
     name = "myproject",
     srcs = ["App.java"],
-    _imports = [
-        "com.google.common.primitives.Ints",
-        "java.lang.Exception",
-        "java.lang.String",
+    _imported_packages = [
+        "com.google.common.primitives",
+        "java.lang",
+        "java.lang",
```

Comment:
Query: I thought this was a set, not a list.

Reply:
Good point! Removed (accidental) duplication.